### DISCUSSION TITLE
Client uses a lot more RAM than expected

### DISCUSSION THREAD

**Discussion by Chiffario** [ppy/osu#27777] [Ideas]
### Type

Performance

### Bug description

A friend of mine recently hit an issue where his osu! client hit almost 10 gigabytes of memory usage and started freezing for multiple seconds until a restart. Thought this might be worth reporting

Renderer: DirectX
OS: Windows 10
Hardware (via DxDiag): [DxDiag.txt](https://github.com/ppy/osu/files/14144961/DxDiag.txt)

### Screenshots or videos

![image](https://github.com/ppy/osu/assets/49232762/46322d02-affe-4824-bc83-0cabb6662fa7)

### Version

2024.131.0

### Logs

[compressed-logs.zip](https://github.com/ppy/osu/files/14144928/compressed-logs.zip)

**Comment by EVAST9919** [comment]
If he plays with storyboard enabled - can be a duplicate of https://github.com/ppy/osu/issues/23258

**Comment by bdach** [comment]
report generally needs more information to be useful

**Comment by smoogipoo** [comment]
To start off with, try with a more up to date version of the game.
